content/accumulators.md

```diff
@@ -0,0 +1,106 @@
++++
+title = "Accumulators"
+gh_repo = "chef-web-docs"
+
+[menu]
+  [menu.infra]
+    title = "Accumulators"
+    identifier = "chef_infra/resources/custom_resources/accumulators"
+    parent = "chef_infra/resources/custom_resources"
+    weight = 65
++++
+
+This is an advanced topic. You should have already written a Custom Resource and be familiar with:
+
+- the built-in Chef Infra `template` resource
+- resource `[run_context](dsl/run_context.md)`
```

Comment:
Should this be `with_run_context` from the [cookbook execution](https://docs.chef.io/infra_language/cookbook_execution/#with_run_context) document?

```suggestion
- Cookbook execution [`with_run_context`]({{< relref "infra_language/cookbook_execution/#with_run_context" >}})
```